frontends/p4/simplifyDefUse.cpp

```diff
@@ -554,19 +634,31 @@ class FindUninitialized : public Inspector {
 
         auto fields = storage->getField(expression->member);
         reads(expression, fields);
+        if (fields == nullptr || fields->isEmpty()) {return false;}
```

Comment:
why this change?

Reply:
Apologies in advance, it is a leftover from when I first started out examining the pass; to clear up the logs. I figured if a LocationSet was empty, no uses could be registered (at least in my test examples).